django_rq/utils.py

```diff
@@ -64,5 +64,9 @@ def get_statistics():
             queue_data['started_jobs'] = len(started_job_registry)
             queue_data['deferred_jobs'] = len(deferred_job_registry)
 
+            if RQ_SCHEDULER_INSTALLED:
+                scheduler = get_scheduler(queue.name)
+                queue_data['scheduled_jobs'] = len(scheduler.get_jobs())
```

Comment:
`scheduler.count()` is a lot more efficient if we only need to know the number of scheduled jobs since we don't need to fetch all the jobs from Redis.